chips/lowrisc/src/registers/README.md

```diff
@@ -0,0 +1,16 @@
+# lowRISC Register Definitions
+
+The files in this directory are auto-generated register definitions for
+lowRISC peripherals.  These files were auto-generated from the OpenTitan
+codebase as follows:
+
+```bash
+$ cd $OPENTITAN_TREE
+$ git checkout Earlgrey-M2.5.2-RC0
+$ bazel build //sw/device/tock:tock_lowrisc_registers
+$ tar -C $TOCK_TREE -xvf bazel-bin/sw/device/tock/tock_lowrisc_registers.tar
+```
+
+Note: the existence of a file in this directory does not necessarily mean
+that a tock peripheral implementation exists.  These files are _only_
+the register definitions.
```

Comment:
```suggestion
Note: the existence of a file in this directory does not necessarily mean
that a tock peripheral implementation exists.  These files are _only_
the register definitions. As an estimate of implementation status, entities
exported in `registers/mod.rs` are those with non-zero implementation.
```

I suspect this will generally hold true?

Reply:
This allows such issues as identified by @alistair23 in https://github.com/tock/tock/pull/3587#discussion_r1282022178 to slip through. We should add all of them to the `mod.rs` unconditionally, which unfortunately would be in conflict with that statement.

Reply:
Yep. This wasn't critical, just a nicety.